# file: C:\Users\dsuth\AppData\Roaming\Python\Python312\site-packages\loguru\_better_exceptions.py
# hypothesis_version: 6.137.1

[128, '\x1b[1m{}\x1b[0m', '\x1b[30m\x1b[1m{}\x1b[0m', '\x1b[31m\x1b[1m{}\x1b[0m', '\x1b[32m\x1b[1m{}\x1b[0m', '\x1b[32m{}\x1b[0m', '\x1b[33m\x1b[1m{}\x1b[0m', '\x1b[33m{}\x1b[0m', '\x1b[34m\x1b[1m{}\x1b[0m', '\x1b[35m\x1b[1m{}\x1b[0m', '\x1b[35m{}\x1b[0m', '\x1b[36m\x1b[1m{}\x1b[0m', '\x1b[36m{}\x1b[0m', ' %s ', '(', ')', '+', '+ ', '+-', ',', '-', '->', '.', '...', '.py', ':', ': ', ';', '=', '>', 'False', 'None', 'True', '[', ']', 'arrows', 'ascii', 'basename', 'builtin', 'cause', 'comment', 'constant', 'context', 'dirname', 'exception_type', 'exception_value', 'file', 'function', 'identifier', 'introduction', 'keyword', 'line', 'number', 'operator', 'other', 'platlib', 'platstdlib', 'punctuation', 'purelib', 'stdlib', 'string', 'utf-8', 'value', '{', '{}', '|', '| ', '}', '│', '└']